website_sale_fixed_name_height/views/website_sale_template.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="products_item" inherit_id="website_sale.products_item" priority="99">
+        <xpath
+            expr="//*[hasclass('o_wsale_products_item_title')]"
+            position="attributes"
+        >
+            <attribute
+                name="t-attf-style"
+                separator=" "
+                add="display: -webkit-box;-webkit-box-orient: vertical;-webkit-line-clamp: #{website.row_count};overflow: hidden; height: #{19*website.row_count}px;"
```

Comment:
**thought**: specifying a height in pixels could create issues for accessibility and may not be compatible with user choices when it comes to font size.

IMO it'd be better to use a relative unit, like `em` or even better here: `lh` https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#line_height_units . 

```
height: #{website.row_count}lh;
```

It probably  won't be perfect in all situations but should still work better most of the times. What do you think?

Reply:
Thanks, I'll fix it)